custom/icds_reports/static/icds_reports/mobile/css/map-chart.less

```diff
@@ -114,7 +114,7 @@ a:active, a:hover, a:focus {
 
 // reduce font size of x-axis labels
 .nvd3 text {
-  font-size: 10px;
+  font-size: 12px;
```

Comment:
it looks like this was explicitly overridden based on comment above. can you confirm this doesn't create any issues - e.g. with long location names in other places?

Reply:
Also, is this the default? if so, we could just remove it.

Reply:
Thanks for pointing out. It would have been really bad. I'm reverting it back to previous font size